lib/mongodb.ts

```diff
@@ -3,30 +3,35 @@ import { MongoClient } from 'mongodb';
 const uri = process.env.MONGODB_URI as string; // your mongodb connection string
 const options = {};
 
-let client;
-let clientPromise: Promise<any>;
-
 declare global {
-  var _mongoClientPromise: Promise<any>;
+  var _mongoClientPromise: Promise<MongoClient>;
```

Comment:
The variable isn't defined in production... best make it optional.

```suggestion
  var _mongoClientPromise?: Promise<MongoClient>;
```

Reply:
🤔 I don't think that's a valid syntax here